playground/src/TabPane.tsx

```diff
@@ -96,4 +98,18 @@ export class TabPane extends React.Component<ITabPaneProps, ITabPaneState>  {
     this.setState({ selectedTabIndex: tabIndex });
   }
 
+  private getTabIndexFromUrl(): number {
+    const { tabs } = this.props;
+
+    let index: number = this.DEFAULT_TAB_INDEX;
+
+    tabs.forEach((tab: ITabDefinition, idx: number, _tabs: ITabDefinition[]) => {
```

Comment:
I'd prefer to use [Array#findIndex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) here, but that requires add'tl changes.

Reply:
Maybe it would be simpler to store the array index in the URL instead of the tab title?

---
In reply to: [223574778](https://github.com/Microsoft/tsdoc/pull/95#discussion_r223574778) [](ancestors = 223574778)